api/chaos/experimentation/v1/experimentation.proto

```diff
@@ -74,13 +83,25 @@ message GetExperimentRunDetailsResponse {
 
 // Simple CRUD API for experiments
 service ExperimentsAPI {
+  // Create a new experiment using the provided experiment data. It fails if an experiment run
+  // with a provided run identifier already exists.
   rpc CreateExperiment(CreateExperimentRequest) returns (CreateExperimentResponse) {
     option (google.api.http) = {
       post : "/v1/chaos/experimentation/createExperiment"
       body : "*"
     };
     option (clutch.api.v1.action).type = CREATE;
   }
+  // Create a new experiment using the provided experiment data. It succeeds even if an experiment run
+  // with a provided run identifier already exists. In this case, the rest of request's paylaod is ignored
```

Comment:
maybe say `run_id` to make it super clear which parameter we're talking about?

Reply:
Added a mention telling them to go to `CreateExperimentData` docs to learn about the details